README.rst

```diff
@@ -5,12 +5,9 @@ SageMaker MXNet Container
 SageMaker MXNet Container is an open-source library for making Docker images for using MXNet on Amazon SageMaker.
 For information on running MXNet jobs on Amazon SageMaker, please refer to the `SageMaker Python SDK documentation <https://github.com/aws/sagemaker-python-sdk>`__.
 
-Table of Contents
+Table Of Contents
 -----------------
-
-#. `Getting Started <#getting-started>`__
-#. `Building your Image <#building-your-image>`__
-#. `Running the tests <#running-the-tests>`__
+.. contents:: \
```

Comment:
GitHub doesn't like rendering RST directives nicely sometimes, and I think this is one of those cases. My suggestion would be to instead do:

```rst
-----------------
Table of Contents
-----------------

.. contents::
  :local:
```

also, the "of" should not be capitalized